tutor/commands/config.py

```diff
@@ -230,8 +234,31 @@ def patches_list(context: Context) -> None:
     renderer.print_patches_locations()
 
 
+@click.command(name="edit", help="Edit config.yml of the current environment")
+@click.pass_obj
+def edit(context: Context) -> None:
+    config_file = tutor_config.config_path(context.root)
+
+    if not os.path.isfile(config_file):
+        raise exceptions.TutorError(f"Missing config file at {config_file}")
```

Comment:
```suggestion
        raise exceptions.TutorError(f"Missing config file at {config_file}. Have you run 'tutor local launch' yet?")
```
In most cases, I believe that the file will be missing if and only if the Tutor has never launched Tutor. What do you think about adding a hint for users here?